packages/@webex/webex-core/test/unit/spec/services-v2/service-host.js

```
@@ -0,0 +1,260 @@
+              url: fixture.defaultUri,
+            };
+          });
+
+          it('should generate a new ServiceHost', () => {
+            assert.instanceOf(ServiceHost.polyGenerate(polyFixture), ServiceHost);
+          });
+        });
+
+        describe('#validate()', () => {
+          it('should throw an error when catalog is missing', () => {
+            delete fixture.catalog;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)